base/set.jl

```diff
@@ -641,12 +645,10 @@ julia> replace!(x->2x, Set([3, 6]))
 Set([6, 12])
 ```
 """
-replace!(prednew::Callable, A; count::Integer=typemax(Int)) =
-    replace!(prednew, A, count=clamp(count, typemin(Int), typemax(Int)) % Int)
-
+replace!(prednew::Callable, A::Union{AbstractArray,AbstractDict,AbstractSet};
+         count::Integer=typemax(Int)) =
+             _replace!(prednew, A, clamp(count, typemin(Int), typemax(Int)) % Int)
```

Comment:
Incorrect indentation.

Reply:
Oh, I trusted my editor... the last line should be only 4 spaces indentation in total?

Reply:
Right.